Makefile

```diff
@@ -35,14 +35,15 @@ IMPT_IMG_NAME=cdi-$(IMPORTER)
 GIT_USER=$(shell git config --get user.email | sed 's/@.*//')
 TAG=$(GIT_USER)-latest
 
-.PHONY: controller importer controller-bin importer-bin controller-image importer-image push-controller push-controller-release push-importer-release push-importer clean test
-all: clean test controller importer
-pre-release: all
+.PHONY: controller importer controller-bin importer-bin controller-image importer-image push-controller push-controller-release push-importer-release push-importer lib clean test
+all: clean test controller importer lib
+#pre-release: all #why??
```

Comment:
I _think_ that maybe the travis script relies on this, which still doesn't make it right.  Feel free to remove it, but check .travis.yaml if you do.